code/game/objects/structures/morgue.dm

```diff
@@ -471,6 +471,8 @@ GLOBAL_LIST_EMPTY(crematoriums)
 		for(var/mob/living/M in conts)
 			if(M.incorporeal_move) //can't cook revenants!
 				continue
+			if(HAS_TRAIT(M, TRAIT_TEMPORARY_BODY)) // you can't cook code
+				continue
```

Comment:
Why are we blocking temp bodies from being cremated? This comment isn't very descriptive

Reply:
Stops people from springing themselves from the Torment Nexus via the crematorium as it skips death code and goes directly to deleting the mob.

Reply:
Noted, maybe leave that comment there rather than `you can't cook code`